cartography/intel/github/repos.py

```diff
@@ -363,9 +432,33 @@ def load(neo4j_session, common_job_parameters, repo_data):
     load_github_owners(neo4j_session, common_job_parameters['UPDATE_TAG'], repo_data['repo_owners'])
     load_github_languages(neo4j_session, common_job_parameters['UPDATE_TAG'], repo_data['repo_languages'])
     load_collaborators(neo4j_session, common_job_parameters['UPDATE_TAG'], repo_data['repo_collaborators'])
+    load_python_requirements(neo4j_session, common_job_parameters['UPDATE_TAG'], repo_data['python_requirements'])
 
 
 @timeit
+def load_python_requirements(neo4j_session, update_tag, requirements_objects):
+    query = """
+    UNWIND {Requirements} AS req
+        MERGE (lib:PythonLibrary:Dependency{id: req.id})
+        ON CREATE SET lib.firstseen = timestamp(),
+        lib.name = req.name
+        SET lib.url = req.url,
+        lib.lastupdated = {UpdateTag}
+
+        WITH lib, req
+        MATCH (repo:GitHubRepository{id: req.repo_url})
+        MERGE (repo)-[r:REQUIRES]->(lib)
+        ON CREATE SET r.firstseen = timestamp()
+        SET r.lastupdated = {UpdateTag},
+        r.specifier = req.specifier
```

Comment:
@aneeshusa I moved the specifier from the PythonLibrary node to the `:REQUIRES` edge. Let me know if you think if there's a better name for it. 

Right now if the specifier is a pinned version, we trim the leading `==`. If the specifier is something more complicated, we leave the whole string in there. If the specifier does not exist, we leave it as None.